tools/commands.py

```diff
@@ -51,7 +51,7 @@ def package_manager(config, commands, catch=False, force=False, default_git_bran
                 force = True
                 commands.pop(commands.index(TENTACLES_FORCE_CONFIRM_PARAM))
             tentacle_manager = TentacleManager(config)
-            tentacle_manager.parse_commands(commands, force=force, default_git_branch=default_git_branch)
```

Comment:
Why removing this ? This means we can't specify the tentacle branch from octobot

Reply:
I shouldn't have commit this. I have a local problem with Tentacle-Manager package.

Reply:
Done 👍